vmdb/app/models/ems_kubernetes.rb

```diff
@@ -1,8 +1,4 @@
 class EmsKubernetes < EmsContainer
-  has_many :container_nodes,                      :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
> We designed these relationship to work also with amazon (groups for sure, services most likely). If we move these to the base class it's easy to (later) discover the problematic ones (if any), on the other hand if we keep them in the subclass it will be much harder to realize that kubernetes and amazon may have some relationship in common that can be moved to the base.

@simon3z 
I agree they should not stay in EmsKubernetes.
But, it looks like this change is not very relevant to this particular PR, and I am also moving them to a container mixin (per @Fryguy recommendation to work with a mixin) in the openshift provider PR:
https://github.com/abonas/manageiq/blob/openshiftprovidermixin/vmdb/app/models/mixins/container_provider_mixin.rb
BTW iirc, we can indeed map amazon's task to container group, but there are no services in amazon, and we don't know what other container providers are going to land in the future, hence I think it's better to leave the base class as clean as possible.